lib/ansiblelint/constants.py

```diff
@@ -0,0 +1,4 @@
+import os
```

Comment:
Maybe even `_constants.py`?

Reply:
@webknjaz When Ansible team will start doing that, we can do it too. ;)